azure-pipelines-full-tests.yml

```diff
@@ -92,3 +72,99 @@ jobs:
         pythonVersion: '$(python.version)'
         profile: '2018-03-01-hybrid'
         fullTest: true
+
+- job: AutomationFullTestPython36ProfileLatest
+  displayName: Automation Full Test Python36 Profile Latest
+  timeoutInMinutes: 9999
+  strategy:
+    maxParallel: 8
+    matrix:
+      8_1:
+        Target: 8_1
```

Comment:
We can give 'Target' a more meaningful name, like `instance`.

Reply:
I thought `target` was a fixed usage 😇
I think instance is better than shard
Because in fact we do have multiple vms running at the same time